### PR TITLE
Chores/Add SHOW_SALE_LISTINGS to app.json #163776367

### PR DESCRIPTION
**Pull request by akegan**
https://www.pivotaltracker.com/story/show/163776367

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "S3_ACCESS_KEY":{
       "required":true
     },
-    "S3_ACCESS_KEY_ID":{
+    "SHOW_SALE_LISTINGS": {
```
Comment (callaghanc) [inline app.json]: @akegan was there a reason to remove the `S3_ACCESS_KEY_ID` env var instead of placing the `SHOW_SALE_LISTINGS` env var below it?

Comment (akegan) [reply app.json]: fixed!

**Comment by akegan** [comment]
@callaghanc is this good to go now?

**Comment by callaghanc** [comment]
yep!

**Comment by callaghanc** [comment]
approved PR, now just quickly testing on review app before accepting pivotal story

**Comment by callaghanc** [comment]
looks good on the review app, marked the pivotal chore finished and am merging this PR

**Pull request merged**